src/pretix/static/pretixpresale/js/widget/widget.js

```diff
@@ -190,26 +190,48 @@ Vue.component('availbox', {
         + '</div>'
         + '<div class="pretix-widget-availability-available" v-if="!item.require_voucher && avail[0] === 100">'
         + '<label class="pretix-widget-item-count-single-label" v-if="order_max === 1">'
-        + '<input type="checkbox" value="1" v-bind:name="input_name">'
+        + '<input type="checkbox" v-model="amount_selected" :name="input_name">'
```

Comment:
Have you tested this does the right thing? I think ``v-model`` on checkboxes refers to whether they are checked or not, not what value they transmit on HTML form submission (which needs to be ``1``).

Reply:
I'd have to double check this, according to the vuejs docs v-model for checkboxes and radiobuttons uses the checked property and the change event. So I think we'll have to replace this with something like `:value="amount_selected" @change="e => amount_selected = value"` and do some type conversion in the setter of `amount_selected`. That would also remove the need for the unary `+`-operator to convert booleans to integers in ` calculate_buy_disabled()`

Reply:
I've tested and the HTML form submission indeed needs to be an integer. I've updated the MR such that the `value` of the checkboxes is an integer, while using the `checked` property in the vue model.